content/docs/reference/addon-api/addon.self.md

```diff
@@ -154,7 +154,7 @@ In that case, the `reenabled` event will fire.
   </tr>
 </table>
 
-Gets a list of addon IDs enabled, optionally filtered using tags.
+Gets a list of addon IDs enabled, optionally filtered using group name.

```

Comment:
Same as above.

Reply:
But for example me when I was creating my first addon, I didn't know how I can filter, which tags, regex, something like that? But okay I can undo that change

Reply:
You can add a link to https://scratchaddons.com/docs/reference/addon-manifest/#tags

Reply:
Okay, so now everything is done.